extensions/amp-analytics/0.1/transport.js

```diff
@@ -118,13 +131,224 @@ export class Transport {
     xhr.send('');
     return true;
   }
+
+  /**
+   * Sends an Amp Analytics trigger event to a vendor's cross-domain iframe,
+   * or queues the message if the frame is not yet ready to receive messages.
+   * @param {string} request
+   * @param {!Object<string, string>} transportOptions
+   */
+  sendRequestUsingCrossDomainIframe(request, transportOptions) {
+    const frameData = Transport.crossDomainIframes_[transportOptions['iframe']];
+    user().assert(frameData, 'Trying to send message to non-existent frame');
```

Comment:
Done. There are 2 remaining uses of user() in code in this file that I didn't change. Shall I change these to dev() also?